src/Interfaces/stochasticstyles.jl

```diff
@@ -55,19 +57,22 @@ default_style(::Type{T}) where T = StyleUnknown{T}()
 The `CompressionStrategy` controls how a vector is compressed after a step.
 
 ## Default implementation:
+
 * [`NoCompression`](@ref): no vector compression
 
 ## Usage
-A subtype of `CompressionStrategy` can be passed as a keyword argument to the
-constructors for some [`StochasticStyle`](@ref)s. Calling
-`CompressionStrategy(s::StochasticStyle)` returns a relevant subtype. The
-default is [`NoCompression`](@ref).
+
+A subtype of `CompressionStrategy` can be passed as a keyword argument to the constructors
+for some [`StochasticStyle`](@ref)s. Calling `CompressionStrategy(s::StochasticStyle)`
+returns a relevant subtype. The default is [`NoCompression`](@ref).
 
 ## Interface
-When defining a new `CompressionStrategy`, subtype it as
-`MyCompressionStrategy <: CompressionStrategy` and define
-a method for
+
+When defining a new `CompressionStrategy`, subtype it as `MyCompressionStrategy <:
+CompressionStrategy` and define these methods:
+
 * [`compress!(s::MyCompressionStrategy, v)`](@ref compress!)
+* [`compress!(s::MyCompressionStrategy, w, v)`](@ref compress!)
```

Comment:
From what I can tell, this two argument method is not being used, so is it for `PDVecs`?